language/move-stdlib/modules/Event.move

```diff
@@ -75,10 +75,10 @@ module Event {
 
     /// Native procedure that writes to the actual event stream in Event store
     /// This will replace the "native" portion of EmitEvent bytecode
-    native fun write_to_event_store<T: copy + drop + store>(guid: vector<u8>, count: u64, msg: T);
+    native fun write_to_event_store<T: drop + store>(guid: vector<u8>, count: u64, msg: T);
```

Comment:
Ah, I guess this is the source of all the questions around events... We can choose to require `store` here or not. It all depends on the philosophical question of whether you want to view event storage the same as global storage...